### DISCUSSION TITLE
Ignore only keyboard shortcuts or other partial ignores

### DISCUSSION THREAD

**Discussion by heiegg** [rxhanson/RectanglePro-Community#69] [Ideas]
I use one app that has loads of keyboard shortcuts configured. I would like to be able to disable Rectangle Pro's keyboard shortcuts for this app. However, I would like to retain the ability to use window throw for this app. Is there some way to configure this? Thank you.

**Comment by rxhanson** [comment]
At the moment, there is not a way to do this. I'll consider adding a terminal flag at least for allowing window throw for disabled apps, though.
